src/atoms/Text.js

```diff
@@ -47,6 +47,18 @@ const Text = styled.p`
     `
     text-align: center;
   `}
+
+  ${props =>
+    props.inline &&
+    `
+    display: inline;
+  `}
+
+  ${props =>
+    props.opacity &&
```

Comment:
The `opacity` prop doesn't mean that the opacity will be set to `0.6.` Try to choose between these two approaches:

## 1. Accept opacity value as a parameter

```jsx
<Text opacity={0.6} />
```

## 2. Use a dedicated `muted` prop:

```jsx
<Text muted /> // means "opacity: 0.6"
```